distribution/tools/plugin-cli/build.gradle

```diff
@@ -38,6 +40,12 @@ tasks.named("dependencyLicenses").configure {
 tasks.named("test").configure {
   // TODO: find a way to add permissions for the tests in this module
   systemProperty 'tests.security.manager', 'false'
+  // These tests are "heavy" on the secure number generator. On Linux, the NativePRNG defaults to /dev/random for the seeds, and
+  // its entropy is quite limited, to the point that it's known to hang: https://bugs.openjdk.org/browse/JDK-6521844
+  // We force the seed to be initialized from /dev/urandom, which is less secure, but in case of unit tests is not important.
+  if (OS.current() == OS.LINUX) {
```

Comment:
I'm wondering if this is something we should set across the board for all tests.

Reply:
Could be worth it, if we observer slowdowns or timeouts. This test might have been an exception as it uses "more" SecureRandom (to check pluigin signatures).

Reply:
I think that's a good idea to avoid potential bottlenecks, and also there's no harm doing so!